tests/baselines/reference/checkJsFiles_skipDiagnostics.errors.txt

```diff
@@ -0,0 +1,53 @@
+tests/cases/compiler/a.js(29,1): error TS2349: This expression is not callable.
+  Type 'Number' has no call signatures.
+tests/cases/compiler/a.js(35,1): error TS2349: This expression is not callable.
+  Type 'Number' has no call signatures.
+
+
+==== tests/cases/compiler/a.js (2 errors) ====
+    var x = 0;
+    
+    
+    /// @ts-ignore
+    x();
+    
+    /// @ts-ignore
+    x();
+    
+    /// @ts-ignore
+    x(
+        2,
+        3);
+    
+    
+    
+    // @ts-ignore
+    // come comment
+    // some other comment
+    
+    // @another
+    
+    x();
+    
+    /* @ts-ignore */
+    /*another comment
+     that could be multiline*/
+    
+    x();

```

Comment:
Yes the question is should this be error or not... Since we are talking about multiline scenarios:: and single line skips the single line comments, should we in general be skipping multiline comments as well

Is better approach to check leading trivia of the node (and or its statement/declaration) where the error is being reported instead?

Reply:
That makes sense to me as an improvement point, thanks. Do you think we could push that into a separate issue? I'm wary of continuing to add cherry picking work for TS 3.9.

Reply:
@DanielRosenwasser and @RyanCavanaugh will have final say but i think not having that should be ok for 3.9 since multiline ts-ignore is new anyway so its not regression. 

Please open new issue to track pending work in that case.
Thanks.

Reply:
https://github.com/microsoft/TypeScript/issues/38370